docs/core/diagnostics/debug-linux-dumps.md

```diff
@@ -0,0 +1,84 @@
+
+### Core dumps with `createdump`
+
+Whereas `dotnet-dump` creates managed-only dumps, [`createdump`](https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/xplat-minidump-generation.md) is the recommended tool for creating core dumps on Linux containing both native and managed information. Other tools like gdb or gcore can also be used to create core dumps but may miss state needed for managed debugging, resulting in "UNKNOWN" type or function names during analysis.
+
+The `createdump` tools is installed with the .NET Core runtime and can be found next to libcoreclr.so (typically in "/usr/share/dotnet/shared/Microsoft.NETCore.App/[version]"). The tool takes a process ID to collect a dump from as its primary argument and can also take optional parameters specifying what kind of dump to collect (a minidump with heap is the default). Options include:
+
+- **`<input-filename>`**
+
+  Input trace file to be converted. Defaults to *trace.nettrace*.
+
+### Options
+
+- **`-f|--name <output-filename>`**
+
+  The file to write the dump to. Default is '/tmp/coredump.%p' where %p is the process ID of the target process.
+
+- **`-n|--normal`**
+
+  Create a minidump.
+
+- **`-h|--withheap`**
+
+  Create a minidump with heap (default).
+
+- **`-t|--triage`**
+
+  Create a triage minidump.
+
+- **`-u|--full`**
+
+  Create a full core dump.
+
+- **`-d|--diag`**
+
+  Enable diagnostic messages.
+
+Note that collecting core dumps requires either the `SYS_PTRACE` capability or that `createdump` be run with sudo or su.
+
+## Analyzing dumps on Linux
+
+Dumps (either managed dumps collected with `dotnet-dump` or core dumps collected with `createdump`) can be analyzed with the `dotnet-dump` tool using the `dotnet-dump analyze` command. As mentioned previously, `dotnet dump` requires that the environment analyzing the dump has the same OS and architecture as the environment the dump was captured in.
+
+Alternatively, lldb can be used to analyze core dumps on Linux, which allows analysis of both managed and native frames. Lldb uses the SOS extension to debug managed code. The [`dotnet-sos`](dotnet-sos.md) CLI tool can be used to install SOS which has [many useful commands](https://github.com/dotnet/diagnostics/blob/master/documentation/sos-debugging-extension.md) for debugging managed code. In order to analyze .NET Core dumps, lldb and SOS require the following .NET Core binaries from the environment the dump was created in:
```

Comment:
What is "lldb"?

Reply:
A native Linux debugger. I'll add a link.